xml/depl_maintenance.xml

```diff
@@ -419,14 +535,239 @@
       &compnode;s needs to be sufficient to evacuate the nodes one by one.
      </para>
     </listitem>
+    <listitem>
+     <para>
+      &o_img; needs to be set up with shared storage; that means the
+      <guimenu>Default Storage Store</guimenu> value in the &o_img; must
+      <emphasis>not</emphasis> be set to <literal>File</literal>.
```

Comment:
This is not exactly true. I've pointed in the bug report to http://docserv.nue.suse.com/documents/Cloud7/suse-openstack-cloud-deployment/html/cha.depl.ostack.html#sec.depl.ostack.glance.ha

which says (I paraprase): **if** there's glance store set up to File, it should be configured as a shared storage.